CONTRIBUTING.md

```diff
@@ -12,3 +14,8 @@ From the root of the project:
 - run `yarn run build`
 - run `yarn run dev`
 
+### Contributing Guidelines
+
+- All hooks must contain example sandboxes and test cases
```

Comment:
Let's expand this better. Something like "When adding a new hook, please make sure that the readme contains usage snippets etc"

Same for the other sentence.

Reply:
pls check again